src/c++/perf_analyzer/client_backend/client_backend.cc

```diff
@@ -95,14 +95,13 @@ BackendKindToString(const BackendKind kind)
 grpc_compression_algorithm
 BackendToGrpcType(const GrpcCompressionAlgorithm compression_algorithm)
 {
-  if (compression_algorithm == COMPRESS_STREAM_GZIP) {
-    return grpc_compression_algorithm::GRPC_COMPRESS_STREAM_GZIP;
```

Comment:
Interesting that they dropped support for STREAM_GZIP.

Reply:
Does this have any backward compatibility impact for the clients? Do we need to relnote this?

Reply:
I think we need to mention this in the relnotes right? More of that we updated grpc than suspended support for compress stream.

Reply:
In GRPC, STREAM_GZIP was marked experimental. I am fine with relnoting it, but this is decision from GRPC and we merely reflect on it.